Jellyfin.Api/Controllers/UserController.cs

```diff
@@ -381,17 +381,13 @@ public async Task<ActionResult<AuthenticationResult>> AuthenticateWithQuickConne
 
             var user = _userManager.GetUserById(userId);
 
-            if (string.Equals(user.Username, updateUser.Name, StringComparison.Ordinal))
-            {
-                await _userManager.UpdateUserAsync(user).ConfigureAwait(false);
```

Comment:
Not needed?

Reply:
UpdatePolicy updates the user, so calling it again is not needed